awd/article.py

```diff
@@ -65,7 +70,7 @@ def __init__(
 
     def process(self) -> None:
         """Run the complete article processing workflow."""
-        self.doi_table.check_doi_and_add_to_table(self.doi)
+        self.doi_process_attempt.check_doi_and_add_to_table(self.doi)
```

Comment:
Didn't catch this last time, but this feels a little awkward.  The rest of these methods are all on the `Article` class, but this is kind of quietly reaching into the `DoiProcessAttempt` model instance and calling a method there.

I think the method `check_doi_and_add_to_table` might make more sense on the `Article` class.